Dragonfly plugin builder failing to install dependency for version v2.0.7-beta.7

### Bug report:

When try to run plugin-builder container its failing to install sqlite dependency

```
/go/src/github.com/glebarez/sqlite/sqlite.go:11:2: code in directory /go/src/github.com/glebarez/go-sqlite expects import "modernc.org/sqlite"
make: *** [Makefile:95: build-dfget] Error 1
```


### Expected behavior:

<!-- Please describe what you expect to happen -->

### How to reproduce it:

```
PLUGIN_TYPE=resource
# custom protocol
PLUGIN_NAME=oras
PLUGIN_PATH=`pwd`/pkg/source/testdata/plugin

docker run --entrypoint=/bin/bash \
  -v "$PLUGIN_PATH":/go/src/plugin \
  -v `pwd`/artifacts/:/artifacts \
  dragonfly-plugin-builder \
  /build.sh "$PLUGIN_TYPE" "$PLUGIN_NAME" /go/src/plugin
```

@embroede 
### Environment:
- Dragonfly version: v2.0.7-beta.7
- OS:
- Kernel (e.g. `uname -a`):
- Others:

--- a/build/plugin-builder/Dockerfile
+++ b/build/plugin-builder/Dockerfile
@@ -10,13 +10,7 @@ RUN apk --no-cache add bash make gcc libc-dev git
 
 COPY . /go/src/d7y.io/dragonfly/v2
 
-# prepare golang dependency
-RUN ls -alh && \
-    go mod vendor && \
-    mv /go/src/d7y.io/dragonfly/v2/vendor/* /go/src/ && \
-    rmdir /go/src/d7y.io/dragonfly/v2/vendor
-
 ENV CGO_ENABLED="1"
-ENV GO111MODULE="off"
+ENV GO111MODULE="on"
 
 COPY build/plugin-builder/build.sh /

--- a/build/plugin-builder/build.sh
+++ b/build/plugin-builder/build.sh
@@ -5,7 +5,7 @@ set -x
 
 export GOPATH=/go
 export CGO_ENABLED="1"
-export GO111MODULE="off"
+export GO111MODULE="on"
 
 cd /go/src/d7y.io/dragonfly/v2 || exit 1
 BUILD_TIME=$(date -u '+%Y-%m-%dT%H:%M:%SZ')

